docs/hub/search.md

```diff
@@ -0,0 +1,15 @@
+# Search
+
+You can now easily search anything on the Hub with **Full-text search**. We index model cards, dataset cards, and Spaces app.py files.
+
+Using the search bar at the top of https://huggingface.co, you can select "Try Full-text search" to find what you seek on the Hub across models, datasets, and Spaces:
+
+![full text search](https://github.com/huggingface/hub-docs/assets/83347449/2a66878d-dc2b-494f-a1f8-4ce12f764892)
+
+![albert](https://github.com/huggingface/hub-docs/assets/83347449/fa5921e8-ad0f-419b-899d-74cb805765f8)
```

Comment:
The images need to be in https://huggingface.co/datasets/huggingface/documentation-images/tree/main/hub and in dark mode as well, see https://github.com/huggingface/hub-docs/blob/main/docs/hub/doi.md?plain=1#L32C1-L35 as an example for this